src/main/java/io/deephaven/csv/CsvSpecs.java

```diff
@@ -117,6 +117,34 @@ public interface Builder {
          */
         Builder headerValidator(Predicate<String> headerValidator);
 
+        /**
+         * True if the input is organized into fixed width columns rather than delimited by a delimiter.
+         */
+        Builder hasFixedWidthColumns(boolean hasFixedWidthColumns);
+
+        /**
+         * When {@link #hasFixedWidthColumns} is set, the library either determines the column widths from the header
+         * row (provided {@link #hasHeaderRow} is set), or the column widths can be specified explictly by the caller.
+         * If the caller wants to specify them explicitly, they can use this method.
+         * 
+         * @param fixedColumnWidths The caller-specified widths of the columns.
+         */
+        Builder fixedColumnWidths(Iterable<Integer> fixedColumnWidths);
```

Comment:
If specified, do all of the column lengths need to be set? All but the last?

Reply:
All but the last need to be set. The last is a placeholder. Made explicit in comments for `CsvSpecs#fixedColumnWidths` and also tested in `CsvReaderTest#finalFixedColumnWidthEntryIsPlaceholder`